Documentation/using-rkt-with-systemd.md

```diff
@@ -189,10 +191,9 @@ $ systemctl status my-socket-activated-app.socket
    Listen: [::]:8080 (Stream)
 
 Jul 30 12:24:50 locke-work systemd[1]: Listening on My socket-activated app's socket.
-Jul 30 12:24:50 locke-work systemd[1]: Starting My socket-activated app's socket.
```

Comment:
It appeared no more when I tried the example, maybe because of a systemd update?